PhysicsTools/NanoAOD/python/custom_jme_cff.py

```diff
@@ -1,7 +1,8 @@
 import FWCore.ParameterSet.Config as cms
 
+from Configuration.Eras.Modifier_run2_jme_2016_cff import run2_jme_2016
+from Configuration.Eras.Modifier_run2_jme_2017_cff import run2_jme_2017
 from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
```

Comment:
From the discussion in this thread, I understand that this import can be removed: please do so, in order to avoid confusion

Reply:
Now removed in the latest commit.